packages/typespec-azure-resource-manager/lib/backcompat.tsp

```diff
@@ -6,8 +6,8 @@ namespace Azure.ResourceManager;
 alias InfrastructureEncryption = CommonTypes.InfrastructureEncryption;
 alias KeyEncryptionIdentity = CommonTypes.KeyEncryptionKeyIdentity;
 alias KeyEncryptionKeyIdentity = CommonTypes.KeyEncryptionKeyIdentity;
-alias CustomerManagedKeyEncryption = CommonTypes.CustomerManagedKeyEncryption;
-alias EncryptionConfiguration = CommonTypes.Encryption;
+alias CustomerManagedKeyEncryption = CommonTypes.customerManagedKeyEncryption;
```

Comment:
Just reiterating the comment above.  This is perpetuating a bug that does not impact the wire API